homework6MethodOverloading/TemperatureConversion.java

```diff
@@ -0,0 +1,39 @@
+package homework6MethodOverloading;
+
+public class TemperatureConversion {
```

Comment:
Temperature Conversion
Write overloaded convertTemperature methods:
- One version accepts a temperature in Celsius and returns it in Fahrenheit.
- The other version accepts a temperature in Fahrenheit and returns it in Celsius

Additional Instructions
Error handling: Handle cases where the input values are inappropriate